package.json

```diff
@@ -14,12 +14,16 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@prisma/client": "^4.13.0",
+    "@types/node": "^18.16.2",
     "express": "^4.18.2",
     "express-graphql": "^0.12.0",
-    "graphql": "^16.6.0",
-    "knex": "^2.4.2",
+    "graphql": "^15.8.0",
     "nodemon": "^2.0.22",
+    "prisma": "^4.13.0",
     "pug": "^3.0.2",
-    "sqlite3": "^5.1.6"
+    "sqlite3": "^5.1.6",
+    "ts-node": "^10.9.1",
+    "typescript": "^5.0.4"
```

Comment:
not sure kung bakit natin sinama sa installation tong typescript stuff, 
    "@types/node": "^18.16.2",
    "ts-node": "^10.9.1",
    "typescript": "^5.0.4"

but i think hindi sila needed. baka we could delete them including tsconfig.json if not needed sila

Reply:
Oh sorry sir I forgot to delete those, nagtest po kasi ako din ng ts dun sa una kong instance. Will remove from dependencies!